xml/System.Windows.Controls/Page.xml

```diff
@@ -73,13 +73,11 @@
   
 -   Programmatically by using <xref:System.Windows.Navigation.NavigationService>.  
   
--   Visually by using the navigation UI of the host, including Windows Internet Explorer, <xref:System.Windows.Navigation.NavigationWindow>, and <xref:System.Windows.Controls.Frame>.  
+-   Visually by using the navigation UI of the host, including the browser, <xref:System.Windows.Navigation.NavigationWindow>, and <xref:System.Windows.Controls.Frame>.  
```

Comment:
This is differentiating between the built in navigation host and a browser host, which IE was as was older Firefox versions.

```suggestion
-   Visually by using the navigation UI of the host, including a browser, <xref:System.Windows.Navigation.NavigationWindow>, and <xref:System.Windows.Controls.Frame>.  
```